src/middleware/validation.js

```diff
@@ -0,0 +1,11 @@
+const validationBody = (schema) => async (req, _, next) => {
+	try {
+		await schema.validateAsync(req.body);

```

Comment:
We installed library express-joi-validation, you do that what is done in a library. Use a library.

Reply:
Ok, I will delete this middleware and use express-joi-validation